gpuflow/dataplane/gpu/dataplane_lpm_gpu.cpp

```diff
@@ -47,7 +47,14 @@ int DataPlaneLPMIPv6GPU::CreateLPMTable() {
     std::cerr << "Create lpm table error from ipv6 lpm factory";  
     exit(1);
   }
-  // TODO: Add ipv6 lpm rules here.
+
+  // Add the rule_0: Direct the Neighbor Solicitation packet to port 0.
+  uint8_t ipv6_addr_0[16];
+  ipv6_addr_0[0] = 255;
+  ipv6_addr_0[1] = 2;
+  ipv6_addr_0[2] = 0;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_0, 24, 0);
+
```

Comment:
How's the Neighbor Solicitation packet work? You can specify `dst_port` in 255 for broadcasting.